src/pages/chanyoung/Main/components/feeds/CommentForm.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+class CommentForm extends React.Component {
+  render() {
+    return (
+      <form className="form" onSubmit={e => this.props.addCommnet(e)}>
```

Comment:
```suggestion
      <form className="form" onSubmit={this.props.addCommnet}>
```
함수를 그대로 넘겨줘도 동일한 결과입니다

Reply:
넵 수정했습니다